main.js

```diff
@@ -0,0 +1,101 @@
+let tituloLivro;
+let capaLivro;
+let statusLeitura;
+let tabela;
+let qtdPaginas;
+let totalPaginas = [];
+let vetorPaginasLidas = [];
+let vetorPaginasNaoLidas = [];
+
+function extraiValores() {
+    tituloLivro = document.querySelector('#titulo');
+    capaLivro = document.querySelector('#capa_livro');
+    statusLeitura = document.querySelector('#status');
+    tabela = document.querySelector('#tabela_livros');
+    qtdPaginas = document.querySelector('#qtd_paginas');
+}
+
+function adicionaLivro() {
+    
+    let linha = tabela.insertRow();
+    let cellLivroLido = linha.insertCell();
+    let cellLivroNaoLido = linha.insertCell(1);
+
+    // Aqui eu tentei fazer a imagem com o Image e o botão de delete com createElement tb, mas aí eu deveria usar 3 appendChild em uma célula da tabela e, pelo o que pesquisei, não tem como.
+    // let novoLivro = new Image(100, 200);
+    // novoLivro.src = capaLivro.value;
+    let novoLivro = "<img src=" + capaLivro.value + ">" + "<br>" + tituloLivro.value;
+ 
+    if (statusLeitura.value == 'sim') {
+        cellLivroLido.innerHTML = novoLivro +
+        "<br><button onclick='deletaLivro(this)'>Delete</button>";
+
+        vetorPaginasLidas.push(Number(qtdPaginas.value));
+    } else {
+        let botaoLido = document.createElement("button");
+        botaoLido.id = "lido";
+        botaoLido.innerHTML = "Lido";
+        cellLivroNaoLido.innerHTML = novoLivro +
+        "<br> <button onclick='deletaLivro(this)'>Delete</button>";
+        cellLivroNaoLido.appendChild(botaoLido); 
+
+        botaoLido.addEventListener('click', () => {
+        marcarComoLido(novoLivro, cellLivroLido)
+        cellLivroNaoLido.remove();
+        })
+
+        vetorPaginasNaoLidas.push(Number(qtdPaginas.value));
+    } 
+
+    contaPaginas(vetorPaginasLidas, vetorPaginasNaoLidas);
+
+}
+
+function deletaLivro(livroAdeletar) {
+    livroAdeletar.parentElement.remove();
+    // Parei aqui: fazendo a contagem de páginas diminuir assim que algum livro for deletado
+    console.log(totalPaginas);
+}
+
+function marcarComoLido(novoLivro, cellLivroLido) {
+    cellLivroLido.innerHTML = novoLivro +
+        "<br><button style='visual.css' onclick='deletaLivro(this)'>Delete</button>";
+}
+
+function contaPaginas(vetorPaginasLidas, vetorPaginasNaoLidas) {
+    
+    let paginasLidas = 0;
+    let paginasNaoLidas = 0;
+    
+
+    for(let i=0; i<vetorPaginasLidas.length; i++) {
+        paginasLidas += vetorPaginasLidas[i];
+    }
+
+    for(let i=0; i<vetorPaginasNaoLidas.length; i++) {
+        paginasNaoLidas += vetorPaginasNaoLidas[i];
+    }
```

Comment:
Nesse caso vc não precisa do For, se tem o total de páginas e o número de páginas lidas é só subtrair um do outro